### PR TITLE
Opensource : 6.2 Compare npm vs yarn

### LINKED ISSUE #2965: Opensource :  6.2  Compare `npm vs yarn`
### Description

<hr>

## Description 📜

Opensource :  6.2  Compare `npm vs yarn`

### NOTE 👇
Please put your doc `6.Popular_Projects` with the name `6.2_npm_vs_yarn.md`

<hr>

### Domain

Open Source

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by iamsunil25**
<hr>

## Opensource : 6.2 Compare npm vs yarn

Fixes #2965   

<hr>

## Compare npm vs yarn📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Open Source 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [x] Open Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [ ] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generates no new warnings.
- [ ] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
image

image

<hr>

### CODE REVIEW

**Comment by iamsunil25** [comment]
@Aryamanz29 when will my pull request merged?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Aryamanz29) [inline .vscode/settings.json:1]: Delete this  `.vscode/settings.json`

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Aryamanz29) [reply .vscode/settings.json:1]: @iamsunil25

Comment (iamsunil25) [reply .vscode/settings.json:1]: okay

Comment (iamsunil25) [reply .vscode/settings.json]: @Aryamanz29 i deleted setting.json file as you said

**Pull request merged**